README.md

```diff
@@ -4,33 +4,40 @@ Open source voice assistant development. Make your own minds!
 
 Open Assistant is a private open source personal assistant prototype able to complete operating system tasks using vocal commands.
 
-# Installation
+## Installation
 
-## Windows
+### Windows
 
-Windows (Click & Run)
+#### _Click & Run_
 
-* Download http://openassistant.org/download/oa_0.21_windows.zip
+* Download [http://openassistant.org/download/oa_0.21_windows.zip](http://openassistant.org/download/oa_0.21_windows.zip)
 * Unzip package
 * Launch `oa.exe` in "Open Assistant" folder
 
-Windows Python Install (Python 3.x)
+#### _Python 3_
```

Comment:
The steps aren't really complete, but this should probably describe running from source in some kind of Windows setup.

```suggestion
#### _Source_
```